language/Values.md

```diff
@@ -175,7 +175,7 @@ Function   | JS Operator | Meaning
 `-`        | `-`         | Numeric subtraction
 `*`        | `*`         | Numeric multiplication
 `/`        | `/`         | Numeric division
-`%`        | `%`         | Numeric modulus
```

Comment:
This isn't quite right actually - `mod` does not behave like `%` in JS, since it returns zero for floats. `purescript-math` does provide JS's `%` operator though.